src/rgw/driver/rados/rgw_tools.cc

```diff
@@ -30,7 +30,7 @@ int rgw_init_ioctx(const DoutPrefixProvider *dpp,
     if (r == -ERANGE) {
       ldpp_dout(dpp, 0)
         << __func__
-        << " ERROR: librados::Rados::pool_create returned " << cpp_strerror(-r)
+        << " ERROR: librados::Rados::pool_create" << pool.name << " returned " << cpp_strerror(-r)
```

Comment:
oops, missing space after `pool_create"` should be `pool_create " <<`

Reply:
Added extra space to the PR